### DISCUSSION TITLE
How do I call a Stored Procedure using EXEC and get the result?

### DISCUSSION THREAD

**Discussion by mayurbojappa** [prisma/prisma#20571] [Q&A]
### Question

I have a stored procedure which accepts parameters and returns a table view.

Query: `EXEC SP_Property @PropertyOptionNo=1,@PropertyNo=10533,@EffectiveDate='8/4/2023'`

When I execute the same using queryRawUnsafe, I get the error 

prisma:error 
Invalid `prisma.$queryRawUnsafe()` invocation:

Raw query failed. Code: `N/A`. Message: `not yet implemented: not yet implemented for SSVariant`

Please let me know how to fix this error.

When i execute it using queryRaw, I get the below error

prisma:error The name '
      EXEC SP_Property @PropertyOptionNo=1,@PropertyNo=10533,@EffectiveDate='8/4/2023'
      ' is not a valid identifier.
library.js:111
prisma:query @P1
library.js:111
prisma:error 
Invalid `prisma.$queryRaw()` invocation:

Raw query failed. Code: `203`. Message: `The name '
      EXEC SP_Property @PropertyOptionNo=1,@PropertyNo=10533,@EffectiveDate='8/4/2023'
      ' is not a valid identifier.`

- OS: Windows 10
- Database: MySQL
- Node.js version: v18.16.0
- Prisma: 4.13.0

**Comment by ludralph** [comment]
Hi @mayurbojappa 👋 

Thank you for raising this question.

Can you please share the relevant parts of your schema file and the raw query you are performing?

**Reply by mayurbojappa** [reply]
Hi @ludralph , although I cannot give you the entire schema, the stored procedure is returning a view with 1 column of datatype sql_variant. I assume this is the issue as it is not able to cast that.

Is there a way to cast the result before it throws an error?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
